File: Java/12EJ-Strings/src/main1.java
import java.util.Scanner;

// 1.	Ingresar una serie de nombres por teclado hasta
// que se cargue la palabra Fin, y mostrar cuántos nombres se ingresaron.

public class main1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Introduce nombres, cuando termine, introduzca 'Fin': ");
        boolean loop = true;
        String nombre = "";
        int i = 0;
        for(;!nombre.equals("Fin"); i++) {
            nombre = sc.next();
        }
        System.out.println("Has introducido " + (i-1) + " nombres.");
    }
}

File: C++/Entrega 4/main1.cpp
#include <stdio.h>
#include <string.h>

int main() {
    int NUM_NOMBRES = 10;

    char nombres[10][50];
    int i, j;

    printf("Introduce 10 nombres.\n");

    for (i = 0; i < NUM_NOMBRES; i++) {
        printf("Nombre %d: ", i + 1);
        scanf("%49s", nombres[i]);
    }

    for (i = 0; i < NUM_NOMBRES; i++) {
        printf("Vocales de '%s': ", nombres[i]);

        for (j = 0; nombres[i][j]; j++) {
            char c = nombres[i][j];

            if (c=='a'||c=='e'||c=='i'||c=='o'||c=='u'||c=='A'||c=='E'||c=='I'||c=='O'||c=='U') {
                printf("%c", c);
            }
        }
        printf("\n");
    }

    return 0;
}